### ISSUE TITLE
`bento/centos-7.7` spins up incorrect CentOS version

### ISSUE THREAD

**Issue by ian28223** [chef/bento#1305]
# Describe the problem
Reopen #1303 1303

I spun up a box using `bento/centos-7.7` and found it spun up 7.8 instead. If I needed 7.8, I would have used `bento/centos-7.8` but I needed 7.7.

```
box 'bento/centos-7.7' (v202005.12.0) for provider: virtualbox
```

```
[root@centos77 vagrant]# cat /etc/*release
CentOS Linux release 7.8.2003 (Core)
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

CentOS Linux release 7.8.2003 (Core)
CentOS Linux release 7.8.2003 (Core)
```

Below are working as expected:

- `bento/centos-7.8` -  spins up a Centos 7.8
- `bento/centos-7.6` -  spins up a Centos 7.6
- `bento/centos-7.5` -  spins up a Centos 7.5
- `bento/centos-7.4` - spins up a Centos 7.4
- `bento/centos-7.3` - spins up a Centos 7.3
- and so on.

Only `bento/centos-7.7` is not spinning up the appropriate CentOS version.

## Stacktrace
na
</details>

**Comment by thibgc** [comment]
I have experienced the same thing; `bento/centos-7.7` results in `CentOS Linux release 7.8.2003 (Core)`.

**Comment by tas50** [comment]
It looks like 7.7 upgraded itself to 7.8 during the build. You can try an older version of that particular CentOS 7 box, but since these are already published and we've moved onto 7.8 there isn't really anything I can do to fix this. CentOS is weirdly statically versioned, but rolling in package nature. This tends to happen when we build.
